workflow/rules/datavzrd.smk

```diff
@@ -71,7 +84,36 @@ rule render_datavzrd_config:
         labels=get_heterogeneous_labels(),
         oncoprint_sorted_datasets="results/tables/oncoprints/{batch}.{event}/label_sortings/",
     log:
-        "logs/datavzrd_render/{batch}.{event}.log",
+        "logs/datavzrd_render/{batch}.{event}.variants.log",
+    template_engine:
+        "yte"
+
+
+rule render_datavzrd_fusions_config:
+    input:
+        template=workflow.source_path(
+            "../resources/datavzrd/fusion-calls-template.datavzrd.yaml"
+        ),
+    output:
+        "resources/datavzrd/{batch}.{event}.fusions.datavzrd.yaml",
+    params:
+        groups=lambda wc: get_report_batch(wc, "fusions"),
```

Comment:
Wrap this so that the lambda is not necessary. Let get_report_batch take the type as input and return a function that takes wildcards as input.